apps/arm/include/WristConfig.h

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#include "ActuatorControllerManager.h"
+#include "Current.h"
+#include "DCMotor.h"
+#include "LookupTable.h"
+#include "OpenLoop.h"
+#include "PID.h"
+#include "Pololu37D.h"
+#include "Position.h"
+#include "Velocity.h"
+#include "hw_bridge.h"
+
+namespace Wrist {
+
+namespace Internal {
+
+static Encoder::Pololu37D leftEncoder({ENC_QUAD_WRST_LHS_A, ENC_QUAD_WRST_LHS_B, NC, 0,
+                                       GPIO::QEI::Encoding::X2_ENCODING});
+static Encoder::Pololu37D rightEncoder({ENC_QUAD_WRST_RHS_A, ENC_QUAD_WRST_RHS_B, NC, 0,
+                                        GPIO::QEI::Encoding::X2_ENCODING});
+
+static Actuator::DCMotor leftMotor(MTR_PWM_WRST_LHS, MTR_DIR_WRST_LHS, false);
+static Actuator::DCMotor rightMotor(MTR_PWM_WRST_RHS, MTR_DIR_WRST_RHS, false);
+
+/* do not use wrist limit switches since joints are individually addressable */
```

Comment:
would like to get feedback on this

Reply:
Can you give more context?

Reply:
uhh i just changed this as a todo to think abt in the future. so wrist has limit switches that only makes sense if u consider both the motors making up the wrist at the same time (might be wrong here). and we've just decided that fw only considers each motor of the claw independetnly. SW has to figure out how to move each motor to make the gap distance x cm.

Reply:
~~~~